src/components/table.js

```diff
@@ -289,3 +300,54 @@ export const TextCell = props => {
 export const HeaderCell = props => {
   return h(TextCell, _.merge({ style: { fontWeight: 500 } }, props))
 }
+
+export const Sortable = ({ sort, field, onSort, children }) => {
+  return div({
+    style: { flex: 1, display: 'flex', alignItems: 'center', cursor: 'pointer', position: 'relative', width: '100%' },
+    onClick: () => onSort(Utils.nextSort(sort, field))
+  }, [
+    children,
+    sort.field === field && div({
+      style: { color: Style.colors.secondary, position: 'absolute', right: 0 }
```

Comment:
Instead of having this be absolutely positioned, what about leaving this in the flex row, with `margin-left: auto` or something like that so it right-aligns, but still takes up space.